clients/cli/src/utils.rs

```diff
@@ -183,3 +187,59 @@ pub fn print_last_successful_sync() {
         println!("Last successful sync: {}", duration);
     }
 }
+
+pub fn set_up_auto_save(
+    watch_file_metadata: FileMetadata,
+    watch_file_location: String,
+) -> Hotwatch {
+    let mut hot_watch = Hotwatch::new_with_custom_delay(core::time::Duration::from_secs(5))
+        .unwrap_or_else(|err| {
```

Comment:
Wonder if we need to fail in this scenario? Maybe this can return an `Option<Hotwatch>` and `stop_auto_save` only gets called if it's not `None`